Makefile.include

```diff
@@ -3,6 +3,10 @@
 
 all:
 
+all-debug: export CFLAGS += -g -O0
```

Comment:
I seem to remember that there was a problem with DWARF on iotlab-m3 way back. @thomaseichinger do you remember what it was and if this is still an issue?